.agents/skills/python-design-patterns/SKILL.md

```
@@ -0,0 +1,411 @@
+        data = CreateUserInput(**(await request.json()))
+        user = await self._service.create_user(data)
+        return Response(user.to_dict(), status=201)
+```
+
+Now HTTP changes don't affect business logic, and vice versa.
+
+### Pattern 3: Separation of Concerns
+
+Organize code into distinct layers with clear responsibilities.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)